components/src/core/components/Input/Select/SelectInputButton.vue

```diff
@@ -6,18 +6,19 @@
     <oxd-button
       class="dropdown-btn"
       :class="dropdownButtonClasses"
-      :label="buttonData.label"
-      :iconName="buttonData.iconName"
+      :label="!(modelValue && modelValue.id > -1) ? buttonData.label : null"
+      :iconName="
+        !(modelValue && modelValue.id > -1) ? buttonData.iconName : null
+      "
       :iconSize="buttonData.iconSize"
       :iconStyle="buttonData.iconStyle"
       :hide-dropdown-label="hideDropdownLabel"
       :size="buttonData.size"
       :style="buttonData.style"
       :displayType="buttonData.displayType"
-      :tooltip="tooltip"
+      :tooltip="$vt(tooltip)"
       :flow="flow"
       @blur="onBlur"
-      @click="onToggleDropdown"
```

Comment:
Check with mike that this behaviour is correct with mike. Now button click wouldn't;t open the dropdown

Reply:
Discussed; As per Mike the button is not just an overlay and it's not clickable and only the dropdown button is clickable